targets/XPlatCppSdk/source/testapps/cppLinuxTestApp/main.cpp

```diff
@@ -15,7 +14,7 @@ void OnPlayFabFail(const PlayFab::PlayFabError& error, void*)
 
 void OnProfile(const PlayFab::ClientModels::GetPlayerProfileResult& result, void*)
 {
-    printf(("========== PlayFab Profile Success: " + result.PlayerProfile->PublisherId + "\n").c_str());
+    printf(("========== PlayFab Profile Success: " + result.PlayerProfile->PlayerId + "\n").c_str());
```

Comment:
>PlayerId [](start = 75, length = 8)

I changed it for better visibility that we received something, because PublisherId was returning an empty string in this test